incubator/hnc/pkg/reconcilers/hnc_config.go

```diff
@@ -222,7 +221,14 @@ func (r *ConfigReconciler) syncObjectReconcilers(ctx context.Context, inst *api.
 // syncActiveReconcilers syncs object reconcilers for types that are in the Spec. If an object reconciler exists, it sets
 // its mode according to the Spec; otherwise, it creates the object reconciler.
 func (r *ConfigReconciler) syncActiveReconcilers(ctx context.Context, inst *api.HNCConfiguration) error {
+	// exist keeps track of existing types in the `config` singleton.
+	exist := map[schema.GroupVersionKind]bool{}
```

Comment:
Maybe make a type out of this since you mention it multiple times?
`type gvkSet map[schema.GroupVersionKind]bool`?

Reply:
Sure! Thanks for the suggestion!